tracer/src/Datadog.Trace/AppSec/Waf/NativeBindings/DdwafConfigStruct.cs

```diff
@@ -4,6 +4,7 @@
 // </copyright>
 
 using System;
+using Datadog.Trace.AppSec.WafEncoding;
```

Comment:
This is the only change of the file, is it required?

Reply:
yes fixed in [6dd3d0e](https://github.com/DataDog/dd-trace-dotnet/pull/4884/commits/6dd3d0eed0ede7f95fda2eed4301f6cdaccb9138)